src/import/FSHImporter.ts

```diff
@@ -125,6 +137,18 @@ export class FSHImporter extends FSHVisitor {
     });
   }
 
+  visitInstance(ctx: pc.InstanceContext) {
+    const instance = new Instance(ctx.SEQUENCE()[0].getText())
+      .withLocation(this.extractStartStop(ctx))
+      .withFile(this.file);
+    instance.instanceOf = ctx.SEQUENCE()[1].getText();
```

Comment:
It's possible that the parent might be specified via a previously defined alias, so I think we actually want to use:
```ts
instance.instanceOf = this.aliasAwareValue(ctx.SEQUENCE()[1].getText());
```

Reply:
Ah, that's correct! I'll make that change.